File: code_challenge_solution/c++/65 Factorial.cpp
```cpp
#include <iostream>
using namespace std;

int factorial(int n) {
    if (n <= 1)
        return 1;
    else
        return n * factorial(n-1);
}

int main() {
    int n;
    cout << "Enter a positive integer: ";
    cin >> n;
    
    if (n < 0)
        cout << "Error! Factorial of a negative number doesn't exist.";
    else
        cout << "Factorial of " << n << " is " << factorial(n);
    
    return 0;
}
```

File: code_challenge_solution/c#/65 Factorial.cs
using System;

namespace FactorialCalculate
{
    class Program
    {
        static void Main(string[] args)
        {
            Console.Write("Enter a number: ");
            int number = Convert.ToInt32(Console.ReadLine());

            int factorial = 1;

            for(int i = 1; i <= number; i++)
            {
                factorial *= i;
            }

            Console.WriteLine("Factorial of {0} is {1}", number, factorial);
        }
    }
}